frontend/src/hooks/queryBuilder/useTag.ts

```diff
@@ -61,13 +61,14 @@ export const useTag = (
 	}, []);
 
 	useEffect(() => {
-		setTags(
-			(query?.tagFilters?.items || []).map((obj) =>
-				isInNotInOperator(obj.op)
-					? `${obj.key} ${obj.op} ${obj.value.join(',')}`
-					: `${obj.key} ${obj.op} ${obj.value.join(' ')}`,
-			),
-		);
+		const initialTags = (query?.filters?.items || []).map((ele) => {
+			if (isInNInOperator(getOperatorFromValue(ele.op))) {
+				const csvString = Papa.unparse([ele.value]);
```

Comment:
This doesn't work. This is not the right way to use the `unparse`.

Reply:
@srikanthccv @palashgdev 
https://github.com/SigNoz/signoz/pull/2629/commits/7e1dc7076f147c2b9a22ca990a7f492a9ca9b0b0